src/legacy/ui/public/directives/__tests__/input_focus.js

```diff
@@ -32,6 +33,8 @@ describe('Input focus directive', function () {
   let selectedText;
   const inputValue = 'Input Text Value';
 
+  uiRoutes.enable();
+
   beforeEach(ngMock.module('kibana'));
   beforeEach(
     ngMock.inject(function (_$compile_, _$rootScope_, _$timeout_) {
```

Comment:
just a note: checked the usage of `input-focus`, and just found usage in timelion, so I think this is a good candidate to migrate there (at a later cleanup)